src/freenet/node/simulator/RealNodeRequestInsertTest.java

```diff
@@ -172,8 +176,8 @@ int insertRequestTest() throws CHKEncodeException, InvalidCompressionCodecExcept
         Node randomNode = nodes[node1];
         //Logger.error(RealNodeRequestInsertTest.class,"Inserting: \""+dataString+"\" to "+node1);
         
-        //boolean isSSK = requestNumber % 2 == 1;
-        boolean isSSK = true;
+        boolean isSSK = requestNumber % 2 == 1;
+        //boolean isSSK = true;
```

Comment:
That looks great! I forgot about the ? operator — and this finally gives me a usecase where it provides better readability than the alternatives :)

Thank you!